client/core/trade.go

```diff
@@ -1276,6 +1276,145 @@ func (t *trackedTrade) isSwappable(ctx context.Context, match *matchTracker) (re
 	return false, false
 }
 
+// checkSwapFeeConfirms returns whether the swap fee confirmations should be
+// checked.
+//
+// This method accesses match fields and MUST be called with the trackedTrade
+// mutex lock held for reads.
+func (t *trackedTrade) checkSwapFeeConfirms(match *matchTracker) bool {
+	if match.MetaData.Proof.SwapFeeConfirmed {
+		return false
+	}
+	_, dynamic := t.wallets.fromWallet.Wallet.(asset.DynamicSwapOrRedemptionFeeChecker)
+	if !dynamic {
+		// Confirmed will be set in the db.
+		return true
+	}
+	if match.Side == order.Maker {
+		if match.Status < order.MakerSwapCast {
+			return false
+		}
+	} else {
+		if match.Status < order.TakerSwapCast {
+			return false
+		}
+	}
+	return true
```

Comment:
Normally wouldn't fuss about this, but I think this is far easier to follow:

```go
if match.Side == order.Maker {
	return match.Status >= order.MakerSwapCast
}
return match.Status >= order.TakerSwapCast
```

Reply:
Doing this.